core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-chips/dot-category-field-chips.component.ts

```diff
@@ -0,0 +1,111 @@
+import { ChangeDetectionStrategy, Component, computed, inject, input, signal } from '@angular/core';
+
+import { ButtonModule } from 'primeng/button';
+import { ChipModule } from 'primeng/chip';
+import { TooltipModule } from 'primeng/tooltip';
+
+import { DotMessageService } from '@dotcms/data-access';
+
+import { DotCategoryFieldKeyValueObj } from '../../models/dot-category-field.models';
+
+/**
+ * Represents the Dot Category Field Chips component.
+ *
+ * @export
+ * @class DotCategoryFieldChipsComponent
+ */
+@Component({
+    selector: 'dot-category-field-chips',
+    standalone: true,
+    imports: [ButtonModule, ChipModule, TooltipModule],
+    templateUrl: './dot-category-field-chips.component.html',
+    styleUrl: './dot-category-field-chips.component.scss',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotCategoryFieldChipsComponent {
+    /**
+     *  Represents the variable 'dotMessageService' which is of type 'DotMessageService'.
+     *
+     * @memberof DotCategoryFieldChipsComponent
+     */
+    readonly #dotMessageService = inject(DotMessageService);
+    /**
+     * Represents the variable 'showAll' which is of type 'signal<boolean>'.
+     *
+     * @memberof DotCategoryFieldChipsComponent
+     */
+    $showAll = signal(false);
+    /**
+     * Represents the variable 'max' which is of type 'number'.
+     *
+     * @memberof DotCategoryFieldChipsComponent
+     */
+    $max = input.required<number>({ alias: 'max' });
```

Comment:
set a constant with the default value (8 or 10)

Reply:
`max = input<number>(MAX_CHIPS,{ alias: 'max' });`

Reply:
Great idea! Fixed